climbing_gym/demo/website_view.xml

```diff
@@ -0,0 +1,359 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+  <record id="homepage" model="ir.ui.view">
+    <field name="name">Home</field>
+    <field name="key">climbing_gym.homepage</field>
+    <field name="type">qweb</field>
+    <field name="website_id" ref="website.default_website"/>
+    <field name="arch" type="xml">
+      <t name="Homepage" t-name="website.homepage">
+        <t t-call="website.layout">
+          <t t-set="pageName" t-value="'homepage'"/>
+          <div id="wrap" class="oe_structure">
+            <section class="s_banner o_full_screen_height pt72 o_cc o_cc5 o_colored_level pb232 oe_img_bg o_bg_img_center o_bg_img_origin_border_box" style="background-image: url(&quot;/unsplash/FHNgTEuxyJA/1029/bouldering.jpg?unique=b4aad0d0&quot;) !important;" data-oe-shape-data="{&quot;shape&quot;:&quot;web_editor/Wavy/22_001&quot;,&quot;flip&quot;:[&quot;x&quot;]}" data-snippet="s_banner" data-name="Banner" data-mimetype="image/webp" data-original-id="1028" data-original-src="/unsplash/FHNgTEuxyJA/bouldering.jpg?unique=ede51c16" data-mimetype-before-conversion="image/jpeg" data-resize-width="1920">
```

Comment:
all attachments that are only used in demo should be defined in demo (for now all are in data).
Also, I'd say we should use `climbing_gym.ir_attachment_id` instead of the unsplash url